zorg/buildbot/builders/annotated/fuchsia-linux.py

```diff
@@ -41,7 +41,7 @@ def main(argv):
     buildbot_revision = os.environ.get('BUILDBOT_REVISION', 'origin/main')
 
     cwd = os.getcwd()
-    build_dir = cwd
+    build_dir = os.path.join(cwd, '..', 'llvm-build')
```

Comment:
I'd suggest keeping the build directory under the CWD:
```suggestion
    build_dir = os.path.join(cwd, 'llvm-build')
```